GDAPI/GDAPI/Application/Editors/GuidelineEditor.cs

```diff
@@ -19,6 +19,10 @@ public class GuidelineEditor
         /// <summary>The <seealso cref="GuidelineEditorPreset"/> that is being edited and will be used in guideline editing operations.</summary>
         public GuidelineEditorPreset Preset;
 
+        /// <summary>Initializes a new instance of the <seealso cref="GuidelineEditor"/> class.</summary>
+        /// <param name="level">The level whose <seealso cref="Guideline"/>s to edit.</param>
+        public GuidelineEditor(Level level)
+            : this(level, new GuidelineEditorPreset("New Preset 0")) { }
```

Comment:
Wouldnt this just cause for the guideline editor to not load the level's and potentially overwrite the level's guideline with nothing?